common/src/main/resources/Operator.properties

```diff
@@ -178,6 +178,7 @@ WLSKO-0218=Domain custom resource converter: Name of the converted file. If not
   file name by appending "__converted." and the input file extension to the base name of the input file name.
 WLSKO-0219=Enable overwriting the existing output file, if any.
 WLSKO-0220=Successfully generated upgraded domain custom resource file ''{0}''.
+WLSKO-0221=Domain custom resource conversion failed due to ''{0}''.
```

Comment:
I'm not sure - but it seems to me that all of the webhook related messages -- the old ones and this new one - should ideally be updated to clearly call out their unique origin "WebLogic Domain custom resource conversion webhook" or similar. Not just "conversion" or just "webhook", but perhaps minimally "domain resource conversion webhook"...

Reply:
Changed in [c11a13e](https://github.com/oracle/weblogic-kubernetes-operator/pull/2902/commits/c11a13e7113f1e3c533b7fac583440da62021087).

Reply:
Looks like 0221 is fixed now - left a new comment for the others.